server/app/blocks_test.go

```diff
@@ -13,14 +14,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func isValidSessionToken(token string) bool {
```

Comment:
Is this use anywhere?

Reply:
Ah, no. Was trying a different approach. Will clean this up.